pages/dkp/kommander/1.4/clusters/attach-cluster/generate-kubeconfig/index.md

```diff
@@ -79,3 +79,15 @@ Before importing this configuration, you can verify that it is functional by run
 ```shell
 kubectl --kubeconfig $(pwd)/kommander-cluster-admin-config get all --all-namespaces
 ```
+
+Then, you can use this kubeconfig to:
+
+- Attach a cluster with [no additional networking restrictions][no-network-restrictions]
+- Attach a cluster that [has networking restrictions][with-network-restrictions]
+
+<p class="message--note"><strong>NOTE: </strong>If a cluster has limited resources to deploy all the federated platform services, it will fail to stay attached in the Kommander UI. If this happens, please check if there are any pods that are not getting the resources required.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>If a cluster has limited resources to deploy all the federated platform services, it will fail to stay attached in the Kommander UI. If this happens, check if there are any pods that are not getting the resources required.</p>
```

Reply:
lol - makes sense about axeing the please.